### PR TITLE
No more Extremely Round Teams

### PR DESCRIPTION
**Pull request by KasparoVy**
:cl:
fix: Nuclear Operatives/ERT Members/Abductors will no longer spawn in with genetic quirks.
/:cl:

### CODE REVIEW

**Comment by Crazylemon64** [comment]
Don't forget to clear `overeatduration`

**Comment by Fox-McCloud** [comment]
Abductors need this too.

File: code/game/gamemodes/miniantags/abduction/abduction.dm [hunk 1]
```diff
@@ -116,6 +116,8 @@
 		H.mind.abductor.agent = 1
 		H.mind.abductor.team = team_number
 		H.real_name = team_name + " Agent"
+		H.reagents.add_reagent("mutadone", 1) //No fat/blind/colourblind/epileptic/whatever ERT.
```
Comment (tigercat2000) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: >copypasted the ert comment too

Comment (KasparoVy) [inline code/game/gamemodes/miniantags/abduction/abduction.dm]: Extra Round Terrestrials

**Pull request merged**